views/pages/data.ejs

```diff
@@ -4,22 +4,32 @@
 <body>
 <% include ../includes/header.ejs %>
 
-<section class="section is-small">
+<section class="section is-medium">
```

Comment:
this add too much space between "Voir les données disponibles" link and "Revenir sur la page d'accueil" button.